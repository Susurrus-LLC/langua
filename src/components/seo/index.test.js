import React from 'react'
import ReactDOM from 'react-dom'
import renderer from 'react-test-renderer'

import { PureSEO as SEO } from '.'

describe('SEO', () => {
  const data = {
    site: {
      siteMetadata: {
        title: 'Langua',
        description: 'Description',
        author: 'Ian A. Cook',
        canonical: 'http://localhost/test'
      }
    }
  }

  it('renders without crashing', () => {
    const div = document.createElement('div')
    ReactDOM.render(<SEO data={data} />, div)
    ReactDOM.unmountComponentAtNode(div)
  })

  it('renders accurately', () => {
    let tree = renderer.create(<SEO data={data} />)
    expect(tree).toMatchSnapshot()
  })
})
