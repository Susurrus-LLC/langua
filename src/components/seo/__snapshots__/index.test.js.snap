// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SEO renders accurately 1`] = `null`;
